pkg/asset/cluster/metadata.go

```diff
@@ -45,23 +45,21 @@ func (m *Metadata) Generate(parents asset.Parents) (err error) {
 	installConfig := &installconfig.InstallConfig{}
 	parents.Get(clusterID, installConfig)
 
-	if installConfig.Config.Platform.None != nil {
-		return nil
-	}
-
 	metadata := &types.ClusterMetadata{
 		ClusterName: installConfig.Config.ObjectMeta.Name,
 		ClusterID:   clusterID.UUID,
 		InfraID:     clusterID.InfraID,
 	}
 
+	platform := installConfig.Config.Platform
```

Comment:
wdyt switching this switch to use platform.Name like https://github.com/openshift/installer/pull/1458/files#diff-bfad726be2a317f0c2bee99d4d62e23eR77

Reply:
Sure